nav-app/src/app/techniques-search/techniques-search.component.html

```diff
@@ -1,34 +1,102 @@
 <div class="techniques-search">
-    <div class="query">
-        <mat-form-field style="width:100%; margin-top: 10px; box-sizing: content-box;">
-            <input matInput [(ngModel)]="query" placeholder="Search Techniques">
-        </mat-form-field>
+    <mat-form-field appearance="outline">
+        <mat-label>Search Techniques</mat-label>
```

Comment:
Label here should not be "search techniques" since it's searching all object types, not just techniques. Probably just "search" would be appropriate?